src/lib/features/authenticate/ui/specs/sign-out-button.spec.ts

```diff
@@ -0,0 +1,16 @@
+import SignOutButton from '../sign-out-button.svelte';
+import { render } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('../../model/sign-out', () => ({
+  signOut: jest.fn(),
+}));
+
+import { signOut } from '../../model/sign-out';
```

Comment:
thoughts (non-blocking): by the way, Jest hoists the `jest.mock` calls so you don't actually have to import after the mock. But I see why you'd want to do this